drivers/display/display_sdl_bottom.c

```diff
@@ -126,17 +127,37 @@ int sdl_display_read_bottom(const uint16_t height, const uint16_t width,
 	return err;
 }
 
-void sdl_display_blanking_off_bottom(void *renderer, void *texture)
+void sdl_display_blanking_off_bottom(void *renderer, void *texture, void *mutex)
 {
+	int err;
+
+	err = SDL_TryLockMutex(mutex);
```

Comment:
Assuming the mutex lock was just absent here before, and this is fixing that? We should split the into another commit if so. Is there a specific reason we need to lock the mutex here? Or is it just to improve the thread safety of this driver?

Reply:
Tbh this entire thing is not thread safe at all, including the sdl calls, so I should probably revert this

Reply:
Fixed